lang/cs/Org.Apache.REEF.Wake/Remote/Parameters/ConnectionRetryCount.cs

```diff
@@ -19,7 +19,7 @@
 
 namespace Org.Apache.REEF.Wake.Remote.Parameters
 {
-    [NamedParameter("Number of retries for connecting to endpoint", defaultValue: "20")]
+    [NamedParameter("Number of retries for connecting to endpoint", defaultValue: "200")]
```

Comment:
1) This is required to run 1000 nodes (the more nodes there are, the longer timeout we need)
2) We started locking driver while submitting tasks, so evaluator needs more time to report connection success while a 1000 tasks are submitted.